### PR TITLE
Fixing link to portage

### PR DESCRIPTION
**Pull request by nray**
Fixed mesh/state wrapper paths to wonton for portage tests.
Added lapack dependency for portage.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -340,6 +340,15 @@ if(ENABLE_PORTAGE)
   if(NOT Boost_FOUND)
     message( FATAL_ERROR "Boost is needed for Portage" )
   endif()
+
+  find_package(LAPACKE REQUIRED)
+  if (NOT LAPACKE_FOUND)
+    message(FATAL_ERROR "Lapacke is needed Portage")
```
Comment (junghans) [inline CMakeLists.txt]: If you say `REQUIRED`, you don't have to thrown an error message.

File: CMakeLists.txt [hunk 2]
```diff
@@ -340,6 +340,15 @@ if(ENABLE_PORTAGE)
...
+  endif()
+  
+  include_directories(${LAPACKE_INCLUDE_DIRS})
+  add_definitions("-DHAVE_LAPACKE")
```
Comment (junghans) [inline CMakeLists.txt]: There should be a `target_link_library(${LAPACKE_LIBRARIES})` somewhere.

**Comment by charest** [comment]
i actually dont think there is a need for Lapack anymore.  This was in the old burton mesh.

File: CMakeLists.txt (hunk 2 above)
Comment (junghans) [reply CMakeLists.txt]: I would just do:
```cmake
find_package(LAPACKE REQUIRED)
```
and drop `message(FATAL_ERROR ....)`.

**Pull request merged**